documentation/src/main/asciidoc/userguide/chapters/caching/Caching.adoc

```diff
@@ -643,9 +643,140 @@ http://www.ehcache.org/documentation/2.8/integrations/hibernate#optional[Ehcache
+
+==== Multi Node Cluster
+
+When running a JPA/Hibernate in a multi-node environment and enabling Infinispan second-level cache, it is necessary to cluster the second-level cache so that cache consistency can be guaranteed.
+Clustering the Infinispan second-level cache provider is as simple as adding the following properties:
+
+====
+[source, XML, indent=0]
+----
+<!-- Use Infinispan second level cache provider -->
+<property name="hibernate.cache.region.factory_class"
+          value="org.hibernate.cache.infinispan.InfinispanRegionFactory"/>
+====
+
+As with the standalone local mode, at the bare minimum the region factory has to be configured to point to an Infinispan region factory implementation.
+
+However, the default Infinispan configuration used by the second-level cache provider is already configured to work in a cluster environment, so no need to add any extra properties.
+
+The next section focuses on analysing how the default cluster configuration works.
+Changing Infinispan configuration options can be done following the instructions in <<caching-provider-infinispan-config, Configuration Properties>> section.
+
+===== Default Cluster Configuration [[integrations:infinispan-2lc-cluster-configuration]]
+
+Infinispan second-level cache provider default configuration is designed for multi-node clustered environments.
+The aim of this section is to explain the default settings for each of the different global data type caches (entity, collection, query and timestamps), why these were chosen and what are the available alternatives.
+These are the characteristics of such configuration:
+
+*  For all entities and collections, whenever a new _entity or collection is read from database_ and needs to be cached, _it's only cached locally_ in order to reduce intra-cluster traffic.
+This option cannot be changed.
+
+*  All _entities and collections are configured to use a synchronous invalidation_ as clustering mode.
+This means that when an entity is updated, the updated cache will send a message to the other members of the cluster telling them that the entity has been modified.
+Upon receipt of this message, the other nodes will remove this data from their local cache, if it was stored there.
+This option can be changed to use replication by configuring entities or collections to use `replicated-entity` cache.
+How to change this option is explained in the <<caching-provider-infinispan-config, Configuration Properties>> section.
+
+*  All _entities and collections have initial state transfer disabled_ since there's no need for it.
+
+* Entities and collections are configured with the following eviction settings.
+You can change these settings on a per entity or collection basis or per individual entity or collection type.
+More information in the <<caching-provider-infinispan-config, Configuration Properties>> section below.
+ - Eviction wake up interval is 5 seconds.
```

Comment:
I think repetition here is good.

The local and default cluster configurations are different files, so they could diverge in the future. So being specific about it is fine in my view.

Besides, if you realise, I've tried to rewrite the docu in such way that a user that has a particular use case can find almost all information it needs in a single section. This is not always possible (e.g. how configuration properties work), but I've tried to make sure that those relevant options are mentioned in the use case section.

With this in mind, I think de-duplication here would not be so good. E.g. if i'm a local user, i shouldn't need to go to the local section to find out about some eviction settings.